pkg/kubelet/images/image_gc_manager.go

```diff
@@ -142,16 +167,23 @@ func (im *realImageGCManager) Start() error {
 		}
 	}, 5*time.Minute, wait.NeverStop)
 
+	// Start a goroutine periodically updates image cache.
+	// TODO(random-liu): Merge this with the previous loop.
```

Comment:
`detectImages` also list image, in fact we can image list there. However, it doesn't list images as frequently as we need, so I add a TODO here to consider this later if needed. :)